NitroxPatcher/Patches/Dynamic/Inventory_Pickup_Patch.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using NitroxClient.GameLogic;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class Inventory_Pickup_Patch : NitroxPatch, IDynamicPatch
+{
+    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((Inventory t) => t.Pickup(default, default));
+
+    public static void Prefix()
+    {
+        Resolve<Items>().InventoryPickingUp = true;
+    }
+
+    public static void Finalizer()
```

Comment:
Could you check if we have other patches where we set a global field. I fear we are using postfixes in them...

Reply:
Couldn't find any other